app/models/entry.server.ts

```diff
@@ -50,6 +50,29 @@ export function getEntries(skip?: number, take: number = 20) {
   })
 }
 
+export function getEntriesByInitialLettersAndPage(
+  initialLetters: string,
+  page: string
+) {
+  const pageNumber: number = parseInt(page);
+  if (isNaN(pageNumber)) {
+    throw new Error(
+      `Page Number ("${page}") must be a number`
+    )
+  } else if (isNonPositive(pageNumber)) {
+    throw new Error(
+      `Page Number ("${page}") must be greater than zero`
+    )
+  }
+
+  const skip: number = (pageNumber - 1) * 100;
+  return getEntriesByInitialLetters(initialLetters, skip);
+}
+
+function isNonPositive(pageNumber: number) {
```

Comment:
Ooh good idea. Also, I'll leave the threads unresolved in case you have more to add? And then you can resolve them if things look good.

Reply:
Sure. It would be helpful to me if you just add a "Done" comment to each little thread once you've committed something that resolves the comment, so I know you've seen it. Thank you!

Reply:
Done!